### ISSUE TITLE
Another incompatible issue

### ISSUE THREAD

**Issue by RockmanZheng** [RockmanZheng/y-intercept#19]
Even #17 is fixed. There is still remaining incompatible issues.

**Comment by RockmanZheng** [comment]
This issue is incorrect. I did not change the test inputs accordingly.

**Comment by RockmanZheng** [comment]
Even I fixed the test input. The stock embedding still seems incompatible. So the input data (excluding sector) is of shape (n_stock, dim-1, period). I expect the output to be something like (n_stock, period, emb_dim). But what we got is (n_stock, emb_dim). Fix this.

**Comment by RockmanZheng** [comment]
The `keras.Dense` layer treats the last dimension as the input dimension. One solution is to loop over all stocks and all periods and use the `keras.Concatenate` layer to construct the embedding output.

**Comment by RockmanZheng** [comment]
This way is too complicated. Actually `keras.Dense` can automatically handle tensor with rank higher than 2. You just need to make sure the last dimension is the input dimension.

**Comment by RockmanZheng** [comment]
Done. See #20
